### PR TITLE
CloudWatch: Adds more examples to CloudWatch Logs cheatsheet

### PR DESCRIPTION
**Pull request by kaydelaney**
**What this PR does / why we need it**:
Adds more examples to the CloudWatch Logs cheatsheet.

### CODE REVIEW

File: public/app/plugins/datasource/cloudwatch/syntax.ts [hunk 1]
```diff
@@ -339,7 +339,7 @@ const tokenizer: Grammar = {
     alias: 'function',
   },
   function: {
-    pattern: new RegExp(`\\b(?:${FUNCTIONS.map(f => f.label).join('|')})`, 'i'),
+    pattern: new RegExp(`\\b(?:${FUNCTIONS.map(f => f.label).join('|')})\\b`, 'i'),
```
Comment (aocenas) [inline public/app/plugins/datasource/cloudwatch/syntax.ts:342]: This change looks a bit unrelated.

**Review by aocenas — APPROVED** [review]

File: public/app/plugins/datasource/cloudwatch/syntax.ts (hunk 1 above)
Comment (kaydelaney) [reply public/app/plugins/datasource/cloudwatch/syntax.ts:342]: Tangentially related in that it fixes the syntax highlighting for the example queries, but I could put it in a separate PR if you think that'd be better
